x-pack/docs/en/security/authorization/role-restriction.asciidoc

```diff
@@ -0,0 +1,26 @@
+[role="xpack"]
+[[role-restriction]]
+=== Role restriction
+
+Role restriction can be used to specify conditions under which a role should be effective.
+Not specifying restriction means a role is not restricted and thus always effective.
+This is the default behaviour.
```

Comment:
We should also describe here the behaviour when restriction is specified but the condition is not met, i.e. when the role is effectively disabled. I am aware this is described in the following "Workflows" section. But I think it makes sense to have a overall statement here at the Restriction level. At workflows level, we can be more specific on what it means by not matching the condition, i.e. not calling the allowed REST APIs. I hope this makes sense.

Reply:
Makes sense. I've tried to reformulate it, but also not repeat myself. Let me know if it reads better now.